homeassistant/components/http/__init__.py

```diff
@@ -346,7 +346,7 @@ def stop(self):
             yield from self.server.wait_closed()
         yield from self.app.shutdown()
         if self._handler:
-            yield from self._handler.finish_connections(60.0)
+            yield from self._handler.shutdown(60.0)
```

Comment:
You think that we should decrease this to 10 seconds or something? 60 seems so long.